test/kongintegration/translator_golden_tests_outputs_test.go

```diff
@@ -108,35 +99,47 @@ func TestTranslatorsGoldenTestsOutputs_Konnect(t *testing.T) {
 
 	for _, goldenTestOutputPath := range allGoldenTestsOutputsPaths(t) {
 		t.Run(goldenTestOutputPath, func(t *testing.T) {
-			ensureGoldenTestOutputIsAccepted(ctx, t, goldenTestOutputPath, updateStrategy)
+			goldenTestOutput, err := os.ReadFile(goldenTestOutputPath)
+			require.NoError(t, err)
+
+			content := &file.Content{}
+			err = yaml.Unmarshal(goldenTestOutput, content)
+			require.NoError(t, err)
+
+			require.Eventually(t, func() bool {
+				if err := updateStrategy.Update(ctx, sendconfig.ContentWithHash{Content: content}); err != nil {
+					t.Logf("error: %v", err)
+					return false
+				}
+				return true
+			}, timeout, tick)
 		})
 	}
 }
 
-func ensureGoldenTestOutputIsAccepted(
-	ctx context.Context,
-	t *testing.T,
-	goldenTestOutputPath string,
-	sut sendconfig.UpdateStrategy,
-) {
+func ensureGoldenTestOutputIsAccepted(ctx context.Context, t *testing.T, goldenTestOutputPath string, kongClient *kong.Client) {
 	goldenTestOutput, err := os.ReadFile(goldenTestOutputPath)
 	require.NoError(t, err)
 
-	content := &file.Content{}
-	err = yaml.Unmarshal(goldenTestOutput, content)
+	cfg := map[string]interface{}{}
```

Comment:
```suggestion
	cfg := map[string]any{}
```
it should be a linter to force it across the codebase. I need to check 😉